src/sonic-config-engine/minigraph.py

```diff
@@ -1432,7 +1432,8 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
     results['MGMT_INTERFACE'] = {}
     mgmt_intf_count = 0
     mgmt_alias_reverse_mapping = {}
-    for key in mgmt_intf:
+    sorted_keys = sorted(mgmt_intf.keys(), key=lambda x : x[0].lower())
```

Comment:
>sorted [](http://example.com/codeflow?start=18&length=6)

Is `sorted` really needed?

Reply:
without sorting, in Py 3 eth1 is returned first, and will be re-written as eth0 in [this line](https://github.com/Azure/sonic-buildimage/pull/11204/files/6351b9a9a5b5bb0daa508842128ce131cd3b4ec5#diff-466ae5de99730b4f3dde9c932e9086536e3a0226858c735fbf776453e8d86fafR1441).

Reply:
Strictly speaking, better to use natural sort, like Management1~Management11

Reply:
changed to use natsort